website/docs/api-backends.md

```diff
@@ -854,6 +854,336 @@ Backpropagate the Mish activation
+| `inplace`   | <tt>bool</tt>     | If `True`, the array may be modified in place. |
+| **RETURNS** | <tt>Floats2d</tt> | The outputs.                                   |
+
+### Ops.backprop_swish {#backprop_swish tag="method"}
+
+<inline-list>
+
+- **default:** <i name="yes"></i>
+- **numpy:** <i name="no"></i>
+- **cupy:** <i name="yes"></i>
+
+</inline-list>
+
+Backpropagate the Swish activation
+[(Ramachandran et al., 2017)](https://arxiv.org/abs/1710.05941v2).
+
+| Argument    | Type              | Description                                         |
+| ----------- | ----------------- | --------------------------------------------------- |
+| `dY`        | <tt>Floats2d</tt> | Gradients of the output array.                      |
+| `X`         | <tt>Floats2d</tt> | The inputs to the forward pass.                     |
+| `Y`         | <tt>Floats2d</tt> | The outputs to the forward pass.                    |
+| `inplace`   | <tt>bool</tt>     | If `True`, the `dY` array may be modified in place. |
+| **RETURNS** | <tt>Floats2d</tt> | The gradient of the input.                          |
+
+### Ops.gelu {#gelu tag="method"}
+
+<inline-list>
+
+- **default:** <i name="yes"></i>
+- **numpy:** <i name="no"></i>
+- **cupy:** <i name="yes"></i>
+
+</inline-list>
+
+. Gelu or "Gaussian Error Linear Unit"
+[(Hendrycks and Gimpel, 2016)](https://arxiv.org/abs/1606.08415) is a
+self-gating non-monotonic activation function similar to [`swish`](#swish):
+whereas [`Gelu`](#gelu) uses the CDF of the Gaussian distribution Φ for
+self-gating `x * Φ(x)` the `swish` activation uses the logistic CDF σ and
+computes `x * σ(x)`. Various approximations exist, but `thinc` implements the
+exact `gelu`. The use of `gelu` is popular within transformer feed-forward
+blocks.
+
+| Argument    | Type              | Description                                    |
+| ----------- | ----------------- | ---------------------------------------------- |
+| `X`         | <tt>Floats2d</tt> | The inputs.                                    |
+| `inplace`   | <tt>bool</tt>     | If `True`, the array may be modified in place. |
+| **RETURNS** | <tt>Floats2d</tt> | The outputs.                                   |
+
+### Ops.backprop_gelu {#backprop_gelu tag="method"}
+
+<inline-list>
+
+- **default:** <i name="yes"></i>
+- **numpy:** <i name="no"></i>
+- **cupy:** <i name="yes"></i>
+
+</inline-list>
+
+Backpropagate `Gelu` activation
+[(Hendrycks and Gimpel, 2016)](https://arxiv.org/abs/1606.08415).
+
+| Argument    | Type              | Description                                         |
+| ----------- | ----------------- | --------------------------------------------------- |
+| `dY`        | <tt>Floats2d</tt> | Gradients of the output array.                      |
+| `X`         | <tt>Floats2d</tt> | The inputs to the forward pass.                     |
+| `inplace`   | <tt>bool</tt>     | If `True`, the `dY` array may be modified in place. |
```

Comment:
```suggestion
| `inplace`   | <tt>bool</tt>     | If `True`, the `dY` array is modified in place. |
```

Otherwise it sounds like it could happen or it could not happen.

Reply:
I added a commit to change this consistently, since the *may be* wording was used for all the new activation functions.